<testsuite name="Mocha Tests" tests="43" failures="5" errors="5" skipped="0" timestamp="Thu, 01 Feb 2018 07:41:39 GMT" time="36.397">
<testcase classname="getService()" name="能正常列出 Bucket" time="1.22"/>
<testcase classname="getAuth()" name="通过获取签名能正常获取文件" time="0.26"/>
<testcase classname="putBucket()" name="正常创建 bucket" time="0.747"/>
<testcase classname="getBucket()" name="正常获取 bucket 里的文件列表" time="0.176"/>
<testcase classname="putObject()" name="fs.createReadStream 创建 object" time="2.148"/>
<testcase classname="putObject()" name="fs.readFileSync 创建 object" time="2.128"/>
<testcase classname="putObject()" name="捕获输入流异常" time="1.107"/>
<testcase classname="getObject()" name="stream" time="2.115"/>
<testcase classname="getObject()" name="body" time="2.114"/>
<testcase classname="sliceUploadFile()" name="正常分片上传 object" time="0.643"/>
<testcase classname="BucketACL" name="putBucketAcl() header ACL:private" time="0.209"/>
<testcase classname="BucketACL" name="putBucketAcl() header ACL:public-read" time="0.276"/>
<testcase classname="BucketACL" name="putBucketAcl() header ACL:public-read-write" time="0.29"/>
<testcase classname="BucketACL" name="putBucketAcl() header GrantRead:uin=&quot;1001&quot;, uin=&quot;1002&quot;" time="0.293"/>
<testcase classname="BucketACL" name="putBucketAcl() header GrantWrite:uin=&quot;1001&quot;, uin=&quot;1002&quot;" time="0.3"/>
<testcase classname="BucketACL" name="putBucketAcl() header GrantFullControl:uin=&quot;1001&quot;, uin=&quot;1002&quot;" time="0.271"/>
<testcase classname="BucketACL" name="putBucketAcl() header ACL:public-read, GrantFullControl:uin=&quot;1001&quot;, uin=&quot;1002&quot;" time="0.302"/>
<testcase classname="BucketACL" name="putBucketAcl() xml" time="0.282"/>
<testcase classname="BucketACL" name="putBucketAcl() xml2" time="0.273"/>
<testcase classname="ObjectACL" name="putObjectAcl() header ACL:private" time="0.273"/>
<testcase classname="ObjectACL" name="putObjectAcl() header ACL:public-read" time="0.207"/>
<testcase classname="ObjectACL" name="putObjectAcl() header ACL:public-read-write" time="0.214"/>
<testcase classname="ObjectACL" name="putObjectAcl() header GrantRead:uin=&quot;1001&quot;, uin=&quot;1002&quot;" time="0.213"/>
<testcase classname="ObjectACL" name="putObjectAcl() header GrantWrite:uin=&quot;1001&quot;, uin=&quot;1002&quot;" time="0.219"/>
<testcase classname="ObjectACL" name="putObjectAcl() header GrantFullControl:uin=&quot;1001&quot;, uin=&quot;1002&quot;" time="0.229"/>
<testcase classname="ObjectACL" name="putObjectAcl() header ACL:public-read, GrantRead:uin=&quot;1001&quot;, uin=&quot;1002&quot;" time="0.244"/>
<testcase classname="ObjectACL" name="putObjectAcl() xml" time="0.254"/>
<testcase classname="ObjectACL" name="putBucketAcl() xml2" time="0.296"/>
<testcase classname="BucketCORS" name="deleteBucketCors()" time="2.077"/>
<testcase classname="BucketCORS" name="putBucketCors(),getBucketCors()" time="2.09"/>
<testcase classname="BucketCORS" name="putBucketCors() old" time="2.085"/>
<testcase classname="BucketCORS" name="putBucketCors() old" time="2.083"/>
<testcase classname="BucketCORS" name="putBucketCors() multi" time="2.086"/>
<testcase classname="BucketTagging" name="putBucketTagging(),getBucketTagging()" time="0"><failure>false == true
AssertionError [ERR_ASSERTION]: false == true
    at test.js:874:13
    at node_modules\cos-nodejs-sdk-v5\sdk\base.js:608:20
    at cb (node_modules\cos-nodejs-sdk-v5\sdk\base.js:1796:13)
    at Request.&lt;anonymous&gt; (node_modules\cos-nodejs-sdk-v5\sdk\base.js:1854:21)
    at IncomingMessage.&lt;anonymous&gt; (node_modules\request\request.js:1085:12)
    at endReadableNT (_stream_readable.js:1062:12)
    at process._tickCallback (internal/process/next_tick.js:152:19)</failure></testcase>
<testcase classname="BucketTagging" name="deleteBucketTagging()" time="0"><failure>false == true
AssertionError [ERR_ASSERTION]: false == true
    at test.js:897:21
    at node_modules\cos-nodejs-sdk-v5\sdk\base.js:559:20
    at cb (node_modules\cos-nodejs-sdk-v5\sdk\base.js:1796:13)
    at Request.&lt;anonymous&gt; (node_modules\cos-nodejs-sdk-v5\sdk\base.js:1854:21)
    at IncomingMessage.&lt;anonymous&gt; (node_modules\request\request.js:1085:12)
    at endReadableNT (_stream_readable.js:1062:12)
    at process._tickCallback (internal/process/next_tick.js:152:19)</failure></testcase>
<testcase classname="BucketTagging" name="putBucketTagging() multi" time="0"><failure>false == true
AssertionError [ERR_ASSERTION]: false == true
    at test.js:912:13
    at node_modules\cos-nodejs-sdk-v5\sdk\base.js:608:20
    at cb (node_modules\cos-nodejs-sdk-v5\sdk\base.js:1796:13)
    at Request.&lt;anonymous&gt; (node_modules\cos-nodejs-sdk-v5\sdk\base.js:1854:21)
    at IncomingMessage.&lt;anonymous&gt; (node_modules\request\request.js:1085:12)
    at endReadableNT (_stream_readable.js:1062:12)
    at process._tickCallback (internal/process/next_tick.js:152:19)</failure></testcase>
<testcase classname="BucketPolicy" name="putBucketPolicy(),getBucketPolicy()" time="0"><failure>false == true
AssertionError [ERR_ASSERTION]: false == true
    at test.js:955:13
    at node_modules\cos-nodejs-sdk-v5\sdk\base.js:465:20
    at cb (node_modules\cos-nodejs-sdk-v5\sdk\base.js:1796:13)
    at Request.&lt;anonymous&gt; (node_modules\cos-nodejs-sdk-v5\sdk\base.js:1854:21)
    at IncomingMessage.&lt;anonymous&gt; (node_modules\request\request.js:1085:12)
    at endReadableNT (_stream_readable.js:1062:12)
    at process._tickCallback (internal/process/next_tick.js:152:19)</failure></testcase>
<testcase classname="BucketPolicy" name="putBucketPolicy() s3" time="0"><failure>false == true
AssertionError [ERR_ASSERTION]: false == true
    at test.js:971:13
    at node_modules\cos-nodejs-sdk-v5\sdk\base.js:465:20
    at cb (node_modules\cos-nodejs-sdk-v5\sdk\base.js:1796:13)
    at Request.&lt;anonymous&gt; (node_modules\cos-nodejs-sdk-v5\sdk\base.js:1854:21)
    at IncomingMessage.&lt;anonymous&gt; (node_modules\request\request.js:1085:12)
    at endReadableNT (_stream_readable.js:1062:12)
    at process._tickCallback (internal/process/next_tick.js:152:19)</failure></testcase>
<testcase classname="BucketLocation" name="getBucketLocation()" time="0.037"/>
<testcase classname="BucketLifecycle" name="deleteBucketLifecycle()" time="2.089"/>
<testcase classname="BucketLifecycle" name="putBucketLifecycle(),getBucketLifecycle()" time="2.093"/>
<testcase classname="BucketLifecycle" name="putBucketLifecycle() multi" time="2.088"/>
<testcase classname="params check" name="Region" time="0"/>
</testsuite>
